app/models/concerns/wakes/redirectors.rb

```diff
@@ -16,7 +16,7 @@ def create_redirect_graph
         return unless canonical_location.present?
         legacy_locations.reload
         legacy_paths_in_redis = legacy_locations.map do |legacy_location|
-          Wakes::REDIS.set(legacy_location.path, canonical_location.path)
+          Wakes::REDIS.set(legacy_location.path, canonical_location.path) unless legacy_location.host.present?
```

Comment:
@chrismaddocks i believe that here you need to worry about a host being set on the `canonical_location`. If it's set, we want to redirect to the url, not the path, right?